tests/test_autoscaler.py

```diff
@@ -0,0 +1,179 @@
+import datetime
```

Comment:
It would be good to get it added to the coveralls but we should add other stuff too
I'll have a look into that.